libs/cardano-ledger-core/src/Cardano/Ledger/CertState.hs

```diff
@@ -312,6 +313,14 @@ data VState era = VState
        )
   , vsDRepDistr :: !(DRepDistr (EraCrypto era))
   , vsCommitteeState :: !(CommitteeState era)
+  -- | Number of contiguous epochs in which there are exactly zero
+  -- active governance proposals to vote on. It is incremented in every
+  -- EPOCH rule if the number of active governance proposals to It
```

Comment:
This sentence looks like it's missing the last part?
`if the number of active governance proposals remains zero` ?